notebooks/225-stable-diffusion-text-to-image/README.md

```diff
@@ -17,7 +17,7 @@ Step-by-step the diffusion process will iteratively denoise latent image represe
 
 The following image shows an example of the input sequence and corresponding predicted image.
 
-**Input text:** cyberpunk cityscape like tokyo nework  with tall buildings at dusk golden hour  cinematic lighting, epic composition. A golden daylight, hyper-realistic environment. Hyper and intricate detail, photo-realistic. Cinematic and volumetric light. Epic concept art. Octane render and Unreal Engine, trending on artstation
+**Input text:** cyberpunk cityscape like Tokyo network  with tall buildings at dusk golden hour  cinematic lighting, epic composition. A golden daylight, hyper-realistic environment. Hyper and intricate detail, photo-realistic. Cinematic and volumetric light. Epic concept art. Octane render and Unreal Engine, trending on artstation
```

Comment:
probably it was New York, not network :)

Reply:
is it a typo or we mean to do so ?

Reply:
it is prompt what used for getting image, I copied it from example with preserving typos and punctuation, as initially provided. I agree that it can misslead, but probably changes should be aligned between notebook and readme.
In this context nework -> New York

Reply:
Thanks for your clarification, I will close this PR